pandas/tseries/tests/test_timeseries.py

```diff
@@ -2235,6 +2235,11 @@ def test_astype(self):
         expected = date_range('1/1/2000', periods=10, tz='US/Eastern').tz_convert('UTC').tz_localize(None)
         tm.assert_index_equal(result, expected)
 
+        # BUG#10442
+        result = pd.Series(pd.date_range('2012-01-01', periods=3)).astype(str)
```

Comment:
add tests with a tz as well

add a short description by the issue comment (e.g. this is astype(str)